languages/python/sqlalchemy-oso/sqlalchemy_oso/roles2.py

```diff
@@ -808,7 +928,7 @@ def _add_query_filter(oso, user, action, resource_model):
         return sql.false()
 
     try:
-        role_list_sql = oso.roles.list_filter_queries[resource_type]
+        role_list_sql = oso.roles.role_allow_list_filter_queries[resource_type]
```

Comment:
Should we rename `_add_query_filter` to `_add_query_role_allow_filter`?

Reply:
yeah probably, gonna just note this for now because I have another branch where I'm restructuring a lot of this sql stuff, will do that there.